app/controllers/events_controller.rb

```diff
@@ -0,0 +1,31 @@
+class EventsController < ApplicationController
+  before_action :authenticate_user!, except: [:index]
+  def index
+    @events = Event.all
+  end
```

Comment:
Index action of events controller should:-

- [ ] list past events, i.e. @past_events = Event.previous_events ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/past_upcoming_events_requirement.png) from Odin)

- [ ] list ucomping events, i.e. @upcoming_events = Event.upcoming_events ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/past_upcoming_events_requirement.png) from Odin)